src/data/shoprows.txt

```diff
@@ -1115,7 +1372,7 @@
 1371	buy	replica emotion chip	Replica Mr. Store
 1372	buy	replica miniature crystal ball	Replica Mr. Store
 1373	buy	replica industrial fire extinguisher	Replica Mr. Store
-1374	buy	replica Jurassic Parka	Replica Mr. Store
+1374	buy	replica Jurassic Parka (dilophosaur mode)	Replica Mr. Store
```

Comment:
mode probably unintentional?

Reply:
Huh. That line was auto-generated in ShopRowData.
That is the result of ```item.toString()```. -> ```item.getName()```:

```
      case ItemPool.JURASSIC_PARKA, ItemPool.REPLICA_JURASSIC_PARKA -> {
        var mode = Preferences.getString("parkaMode");
        yield mode.equals("") ? this.name : this.name + " (" + mode + " mode)";
      }
```
I should use ```item.getDataName()```. Maybe. ```toString()``` includes count, if > 1.
Or generate that file not-logged in. Will standard rewards generate their buy requests then?

I'll look at this in my next PR. Perhaps I can simply edit shoprows.txt for this one.

Reply:
That worked.